dbms/src/Storages/StorageODBC.cpp

```diff
@@ -0,0 +1,63 @@
+#include <Storages/StorageODBC.h>
+#include <Storages/StorageMySQL.h>
+
+#include <Interpreters/Context.h>
+#include <Interpreters/ExpressionAnalyzer.h>
+
+#include <Dictionaries/ODBCBlockInputStream.h>
+
+#include <Parsers/ASTSelectQuery.h>
+
+#include <Common/typeid_cast.h>
+
+namespace DB
+{
+
+/** Implements storage in the ODBC database.
+  * Use ENGINE = odbc(odbc connection string, table name)
+    Example ENGINE = odbc('dsn=test', table)
```

Comment:
Is it Ok to write `dsn=...` instead of `DSN=...`?

Reply:
Is is ok, it case-insensitive